File: m/rls/stupr.r
file	st_upr - uppercase string
include	rid:rider
include	rid:stdef
include	rid:chdef

code	st_upr - uppercase line

  func	st_upr
	str : * char~
	()  : * char
  is	ptr : * char = str
	spin while (*ptr++ = ch_upr (*ptr)) ne
	reply str
  end


File: m/rls/rlb/stupr.c
/* file -  st_upr - uppercase string */
#include "m:\rid\rider.h"
#include "m:\rid\stdef.h"
#include "m:\rid\chdef.h"
/* code -  st_upr - uppercase line */
char *st_upr(
char *str )
{ char *ptr = str;
  while ( (*ptr++ = ch_upr (*ptr)) != 0) {;}
  return ( str);
} 
